### PR TITLE
chore(SpokePool): Remove pre-v3 types and events

### PR DESCRIPTION
**Pull request by pxrl**
These are no longer needed and would potentially otherwise be clobbered by upcoming svm changes. Remove them now to simplify that work.

### CODE REVIEW

File: contracts/test/MockSpokePool.sol [hunk 1]
```diff
@@ -145,247 +145,6 @@ contract MockSpokePool is SpokePool, MockV2SpokePoolInterface, OwnableUpgradeabl
...
-            destinationChainId,
-            relayerFeePct,
-            newDepositId,
-            quoteTimestamp,
-            originToken,
-            recipient,
-            msg.sender,
-            message
-        );
-    }
```
Comment (pxrl) [inline contracts/test/MockSpokePool.sol:178]: We might want to retain this, tbf.

**Comment by pxrl** [comment]
@mrice32 @nicholaspai Are we OK to move on this cleanup? We have a lot of WIP work that would ideally come after it, but those PRs are instead internalising some overlapping changes.

**Review by md0x — APPROVED** [review]
Looks good to me

**Review by nicholaspai — APPROVED** [review]
I think we can merge this

File: contracts/test/MockSpokePool.sol (hunk 1 above)
Comment (pxrl) [reply contracts/test/MockSpokePool.sol]: Restored in a test capacity here: https://github.com/across-protocol/contracts/pull/663/commits/0186b0714467640f93f16882d5ebbf44f393d731

**Review by mrice32 — APPROVED** [review]

**Pull request merged**